masq/src/commands/setup_command.rs

```diff
@@ -338,4 +371,59 @@ ERRORS:
 ip                            No sir, I don't like it.\n\
 \n");
     }
+
+    #[test]
+    fn setup_command_with_data_directory_shows_right_path() {
+        vec![
```

Comment:
With a little imagination, this could probably be made more readable. The formatter directive `#[rustfmt_skip]` might help.

Reply:
#[rustfmt::skip] used :) thanks for tip